doc/releases/v0.3.0.txt

```diff
@@ -11,12 +22,18 @@ Feature-Finding
 
 - The percentile-based thresholding was moved into a serpate function, so it can now be called directly. That will be useful to users who want to inspect what the thresholding is doing to their images. It will also be useful for profiling. (:issue:`139`)
 
+- Various performance improvements make feature-finding and trajectory-linking faster than ever, in spite of the increased functionality. (:issue:`203`, :issue:`204`)
+
+- More user-friendly error messages replace cryptic errors. (:issue:`166`).
+
 Bug Fixes
 ~~~~~~~~~
 
 - Fixed a bug in v0.2.3 and v0.2.4 that broke the ``circle_size`` parameter in ``annotate()``. (:issue:`169`, :issue:`170`)
+- Fixed a bug where subnetwork distances were not addded in quadrature in the python linker. (:issue:`212`). (The impact of this bug is not as large as one might expect. See issue link for details.)
+- Input images with a float datatype are internally converted to integer datatype for faster computaiton. They were converted to signed integers; now they are converted to unsigned. (:issue:`188`)
```

Comment:
- Slow feature-finding with more recent versions of the numba package (0.16 and 0.17 specifically) is addressed.